parser.cs

```diff
@@ -422,7 +465,9 @@ public class Parser {
 
     private void emptyStatement(){
         switch(__lookahead.Type) {
+            case TOKENS.END:
```

Comment:
Not sure if there's more, but this was the only one I found. It's likely there's more but they will stand out during parsing.